lib/common_test/src/test_server_ctrl.erl

```diff
@@ -3910,8 +3928,20 @@ run_test_case1(Ref, Num, Mod, Func, Args, RunInit,
 	{skip_init,_} ->			% conf doesn't count
 	    ok;
 	{_,ok} ->
+		case Loc of
+			[{Module, _}] when Module =/= ct_framework ->
+				put(test_server_total_time, get(test_server_total_time) + Time);
+			_ -> not_a_test_fun end,
 	    put(test_server_ok, get(test_server_ok)+1);
 	{_,failed} ->
+		TTime = case Time of
```

Comment:
TeaTime ;-) ?
Either we should come up with more meaningful names (IMHO preferred) or if not possible stick to variables with numbers (Time0, Time1, Time2 ...) which is more common.